README.md

```diff
@@ -51,8 +51,12 @@ Check out our [installation instructions](documentation/installation.md) for mor
 
 ## Configuration
 
-Git Town is configured on a per-repository basis.
-Upon first use in a repository, it will ask for all required configuration.
+Git Town is configured on a per-repository basis. It requires two pieces of information.
+
+* the main development branch
+* the perennial branches (see more [here](/documentation/development/branch_hierarchy.md#perennial-branches))
```

Comment:
What about making the words "perennial branches" the list to the documentation of perennial branches?